.browserslistrc

```diff
@@ -0,0 +1,3 @@
+> 1%
```

Comment:
Just a comment (not a suggestion): I managed somehow to move some of these configuration files into `package.json`: https://github.com/krzysztofrewak/browary/blob/master/package.json#L59